dmotion:
  ankle_pitch:
    data: [0, 0.95843091334894615, 1.9956436860504094, 2.1261449198120035, 2.1407807781133386, 2.4823565974072102, 3.7587040433846424, 5.5, 0, 0.033691614530089842, 39.491136090729469, -11.237473897459383, -31.76055009999919, -18.182928203434713, -0.080730895207423714, 0, 0, 0.16734260908141946, 0, -114.58865012931754, -22.903765548431426, 38.188459297026277, 0, 0]